apps/wallet/src/ui/app/pages/accounts/manage/ManageAccountsPage.tsx

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { ArrowBgFill16, Plus12 } from '@mysten/icons';
+import * as CollapsiblePrimitive from '@radix-ui/react-collapsible';
+import { useNavigate } from 'react-router-dom';
+import { AccountItem } from '../../../components/accounts/AccountItem';
+
+import { AccountActions } from '../../../components/menu/content/AccountActions';
+import Overlay from '../../../components/overlay';
+import { useAccounts } from '../../../hooks/useAccounts';
+import { Heading } from '../../../shared/heading';
+import { AccountType, type SerializedAccount } from '_src/background/keyring/Account';
+import { Text } from '_src/ui/app/shared/text';
+import { ButtonOrLink, type ButtonOrLinkProps } from '_src/ui/app/shared/utils/ButtonOrLink';
+
+// todo: these will likely be shared in various parts of the UI
+const labels = {
+	[AccountType.DERIVED]: 'Passphrase Derived',
+	[AccountType.LEDGER]: 'Ledger',
+	[AccountType.IMPORTED]: 'Imported',
+	[AccountType.QREDO]: 'Qredo',
+};
+
+// todo: we probbaly have some duplication here with the various FooterLink / ButtonOrLink
+// components - we should look to add these to base components somewhere
+function FooterLink({ children, to, ...props }: ButtonOrLinkProps) {
+	return (
+		<ButtonOrLink
+			className="text-hero-darkest/40 no-underline uppercase group-hover:text-hero"
+			to={to}
+			{...props}
+		>
+			<Text variant="captionSmallExtra" weight="medium">
+				{children}
+			</Text>
+		</ButtonOrLink>
+	);
+}
+
+// todo: this is slightly different than the account footer in the AccountsList - look to consolidate :(
+function AccountFooter() {
+	return (
+		<div className="flex flex-shrink-0 w-full">
+			<div className="flex gap-3">
+				<div className="w-4" />
+				<FooterLink to="/edit">Edit Nickname</FooterLink>
+				<FooterLink to="/remove">Remove</FooterLink>
+			</div>
+		</div>
+	);
+}
+
+function Section({ accounts, type }: { accounts: SerializedAccount[]; type: AccountType }) {
```

Comment:
lets move this and the other components to their own file. We should probably rename this to something like `AccountsGroup` or something like that. We can have multiple account groups with the same type. Like 2 zk with different provider or even same provider but different email.

We might need to pass title as well but we can update that when necessary. Also maybe specific props based on the account type, like mnemonic source id if accounts are mnemonic-derived.